shared/Utils.js

```diff
@@ -3,5 +3,57 @@
 export default {
 	stripHtmlTags (inputString) {
 		return inputString.replace(/<\/?[^>]+(>|$)/g, "");
+	},
+
+	/**
+	 * The complete Triforce, or one or more components of the Triforce.
```

Comment:
This should be a specific description of the type you are defining. I think you probably got this from the link I sent, but it was constructed as an example which I believe is actually a reference to a [video game](https://en.wikipedia.org/wiki/Triforce). Although it is somewhat fitting here as well.